### ISSUE TITLE
initGL() is being called multiple times

### ISSUE THREAD

**Issue by benwaffle** [Flafla2/Remote2D-Engine#7]
check the eclipse console

**Comment by Flafla2** [comment]
Yes, that is by design.  InitGL() is called every time the display is reset, or glViewport() is needed - so whenever you resize the window or go into/out of fullscreen.  It could be optimized probably, but in all honesty it really isn't a very expensive operation.
